css/src/components/checkbox.scss

```diff
@@ -0,0 +1,118 @@
+/* stylelint-disable selector-max-specificity, max-nesting-depth, selector-no-qualifying-type */
+
+$checkbox-border-radius: $radius-sm !default;
+$checkbox-border-color: $text !default;
+$checkbox-check-hover-color: $secondary !default;
+$checkbox-icon-size: 0.875rem !default;
```

Comment:
Internal spacing for components should be based on `em` whenever possible, so that we can easily modify sizes with font-size alone.

Reply:
Updated to `em`.